_posts/2019-09-30-release-1.2.1.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: 'Release v1.2.1'
+tags: ['v1.2.1', 'changelog']
```

Comment:
why not add a tag hotfix? Then it would be easier to use search in the future.

Reply:
Good idea!